tensorboard/context_test.py

```diff
@@ -14,32 +14,53 @@
 # ==============================================================================
 """Tests for tensorboard.context."""
 
+import ipaddress
+
 from tensorboard import auth as auth_lib
 from tensorboard import context
 from tensorboard import test as tb_test
 
+REMOTE_IP = ipaddress.ip_address("192.168.0.1")
+X_FORWARDED_FOR_IPS = [ipaddress.ip_address("2001:db8::"), REMOTE_IP]
```

Comment:
Please minimize global mutable state—can this be a tuple?

Reply:
Done.